thumbor/detectors/face_detector/__init__.py

```diff
@@ -25,7 +25,10 @@ def __add_hair_offset(self, top, height):
         return top
 
     def detect(self, callback):
-        features = self.get_features()
+        try:
+            features = self.get_features()
+        except Exception:
+            features = None
```

Comment:
Could you add logging here? Silencing exceptions isn't great. Also added a metric would be awesome so things can be tracked.